cptv-convert.py

```diff
@@ -54,6 +65,16 @@ def make_mp4(cptv_file, output_folder, colormap):
             thermal_image = thermal_image.resize((int(thermal_image.width * FRAME_SCALE), int(thermal_image.height * FRAME_SCALE)), Image.BILINEAR)
             mpeg.next_frame(np.asarray(thermal_image))
         mpeg.close()
+    copy_file(mp4_temp, mp4_file)
+    os.remove(mp4_temp)
+    if copy and delete:
+        os.remove(cptv_file)
+
+def copy_file(src, dest):
+    copyfile(src, dest)
+    fs = os.open(dest, os.O_RDONLY)
+    os.fsync(fs)
+    os.close(fs)
```

Comment:
Interesting. I didn't know about this approach. It definitely works? (fine if it does!)

Reply:
Yeah, worked in the testing I did :)